cob_obstacle_distance/src/distance_manager.cpp

```diff
@@ -90,7 +90,7 @@ int DistanceManager::init()
         return -1;
     }
 
-    if(!nh_.getParam("joint_names", this->joints_))
+    if(!nh_.getParam("/arm_left/joint_names", this->joints_))
```

Comment:
do not use hardcoded namespaces....
It's ok for testing but in the end should be kept as general as possible...if you need `joint_states` within this library, you could also subscribe to `/joint_states` topic which has __all__ joint_states included....you just would need a proper parsing/mapping strategy to extract the joint_states relevant for this library